index.html

```diff
@@ -0,0 +1,716 @@
+
+        <div class="more-article-single">
+          <figure>
+            <img src="images/thumbnails/seven.jpg" alt="Moon article">
+            <figcaption class="caption-text-color">Jeenah Moon for The New York Times</figcaption>
+          </figure>
+          <h4>A New York Clock That Told Time Now Tells the Time Remaining</h4>
+          <h6 class="caption-text-color font-small">Sept. 21</h6>
+        </div>
+        <div class="more-article-single">
+          <figure>
+            <img src="images/thumbnails/eight.jpg" alt="Moon article">
+            <figcaption class="caption-text-color">Warner Bros.</figcaption>
+          </figure>
+          <h4>How ‘Goodfellas’ and the Gangster Class of 1990 Changed Hollywood</h4>
+          <h6 class="caption-text-color font-small">Sept. 16</h6>
+        </div>
+        <div class="more-article-single">
+          <figure>
+            <img src="images/thumbnails/one.jpg" alt="Moon article">
+            <figcaption class="caption-text-color">John Taggart for The New York Times</figcaption>
+          </figure>
+          <h4>It’s a Banana. It’s Art. And Now It’s the Guggenheim’s Problem.</h4>
+          <h6 class="caption-text-color font-small">Sept. 19</h6>
+        </div>
+      </div>
+
+    </div>
+
+
+    <div class="most-popular-side-bar">
+      <h4>Most Popular</h4>
+      <div class="more-popular-div">
+        <p>
+          Trump Jump-Starts Misinformation on Ginsburg’s ‘Dying Wish’
+        </p>
+        <p>
+          The Biggest Wave Surfed This Year
+        </p>
+        <p>
+          Opinion: An Open Letter to Mitt Romney
+        </p>
+        <p>
+          Trump Could Be Investigated for Tax Fraud, D.A. Says for First Time
+        </p>
+        <p>
+          Opinion: Trump’s Supreme Court Pick May Need to Denounce Roe. Good.
+        </p>
+        <p>
+          Pack a ‘Go Bag’ Now
+        </p>
+        <p>
+          Opinion: Mitch McConnell Has Seen the Promised Land
+        </p>
+        <p>
+          Pandemic Puts N.Y.C. Hotels on the Brink: ‘A Complete Washout’
+        </p>
+        <p>
+          The War Crime No One Wants to Talk About
+
+        </p>
+        <p>
+          Opinion: Was That Ride on the Trump Train Worth It?
+        </p>
+
+      </div>
+    </div>
+
+  </section>
+
+  <footer>
+    <div class="footer-head flex-horizontal">
+      <img src="/images/the-new-york-times-logo.svg" alt="new york times logo">
+      <a class="font-small" href="#">Go to Home Page >></a>
+
+    </div>
+
+    <div class="footer-list flex-horizontal font-small">
+      <div class="flex-vertical">
+        <h4>NEWS</h4>
+        <p>Home Page</p>
+        <p>World</p>
+        <p>U.S</p>
+        <p>Politics</p>
+        <p>Elections 2020</p>
+        <p>New York</p>
+        <p>Business</p>
+        <p>Tech</p>
+        <p>Science</p>
+        <p>Sports</p>
+        <p>Obtuaries</p>
+        <p>Today's paper</p>
+        <p>Corrections</p>
```

Comment:
These items should be linked. You are free to change the behavior with CSS, but you should wrap the items with `<a>` tags.